src/lib/viewers/controls/findbar/FindBarToggle.tsx

```diff
@@ -3,16 +3,24 @@ import IconSearch24 from '../icons/IconSearch24';
 import './FindBarToggle.scss';
 
 export type Props = {
-    onFindBarToggle?: () => void;
+    onFindBarToggle?: (buttonElement: HTMLDivElement | null) => void;
 };
 
 export default function FindBarToggle({ onFindBarToggle }: Props): JSX.Element | null {
+    const buttonReference = React.useRef(null);
+
     if (!onFindBarToggle) {
         return null;
     }
 
     return (
-        <button className="bp-FindBarToggle" onClick={onFindBarToggle} title={__('toggle_findbar')} type="button">
+        <button
+            ref={buttonReference}
+            className="bp-FindBarToggle"
+            onClick={(): void => onFindBarToggle(buttonReference.current)}
```

Comment:
Can we do the following to avoid the ref?

```
onClick={({ target }) => onFindBarToggle(target)}
```

Reply:
of course